src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -9552,6 +9552,7 @@
           # Targeting Spring 2024 Matter release
           - EnergyEVSEMode
           - DeviceEnergyManagementMode
+          - WaterHeaterMode
```

Comment:
This is in the wrong place.  It needs to go down in the "future" section below, not into a past release.

Or just take this out; https://github.com/project-chip/connectedhomeip/pull/34394 fixes this part.

Reply:
Change made.